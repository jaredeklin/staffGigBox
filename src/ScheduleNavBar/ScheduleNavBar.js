import React from 'react';
import { NavLink } from 'react-router-dom';

const ScheduleNavBar = () => {
  return (
    <ul className="header-nav-list">
      <NavLink to="/schedule" exact>
        <li>All Venues</li>
      </NavLink>
      <NavLink to="/schedule/ogden" exact>
        <li>Ogden Theatre</li>
      </NavLink>
      <NavLink to="/schedule/gothic" exact>
        <li>Gothic Theatre</li>
      </NavLink>
      <NavLink to="/schedule/bluebird" exact>
        <li>Bluebird Theater</li>
      </NavLink>
      <NavLink to="/schedule/individual" exact>
        <li>Your Schedule</li>
      </NavLink>
    </ul>
  );
};

export default ScheduleNavBar;
