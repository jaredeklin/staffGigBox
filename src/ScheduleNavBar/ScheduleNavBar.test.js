import React from 'react';
import { shallow } from 'enzyme';
import ScheduleNavBar from './ScheduleNavBar';

describe('ScheduleNavBar', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<ScheduleNavBar />);

    expect(wrapper).toMatchSnapshot();
  });
});
