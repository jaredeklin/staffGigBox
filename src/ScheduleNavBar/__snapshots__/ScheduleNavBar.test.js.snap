// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleNavBar should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ScheduleNavBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          aria-current="page"
          exact={true}
          to="/schedule"
>
          <li>
                    All Venues
          </li>
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          exact={true}
          to="/schedule/ogden"
>
          <li>
                    Ogden Theatre
          </li>
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          exact={true}
          to="/schedule/gothic"
>
          <li>
                    Gothic Theatre
          </li>
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          exact={true}
          to="/schedule/bluebird"
>
          <li>
                    Bluebird Theater
          </li>
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          exact={true}
          to="/schedule/individual"
>
          <li>
                    Your Schedule
          </li>
</NavLink>,
      ],
      "className": "header-nav-list",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <li>
            All Venues
</li>,
          "exact": true,
          "to": "/schedule",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "All Venues",
          },
          "ref": null,
          "rendered": "All Venues",
          "type": "li",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <li>
            Ogden Theatre
</li>,
          "exact": true,
          "to": "/schedule/ogden",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Ogden Theatre",
          },
          "ref": null,
          "rendered": "Ogden Theatre",
          "type": "li",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <li>
            Gothic Theatre
</li>,
          "exact": true,
          "to": "/schedule/gothic",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Gothic Theatre",
          },
          "ref": null,
          "rendered": "Gothic Theatre",
          "type": "li",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <li>
            Bluebird Theater
</li>,
          "exact": true,
          "to": "/schedule/bluebird",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Bluebird Theater",
          },
          "ref": null,
          "rendered": "Bluebird Theater",
          "type": "li",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <li>
            Your Schedule
</li>,
          "exact": true,
          "to": "/schedule/individual",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Your Schedule",
          },
          "ref": null,
          "rendered": "Your Schedule",
          "type": "li",
        },
        "type": [Function],
      },
    ],
    "type": "ul",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            aria-current="page"
            exact={true}
            to="/schedule"
>
            <li>
                        All Venues
            </li>
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            exact={true}
            to="/schedule/ogden"
>
            <li>
                        Ogden Theatre
            </li>
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            exact={true}
            to="/schedule/gothic"
>
            <li>
                        Gothic Theatre
            </li>
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            exact={true}
            to="/schedule/bluebird"
>
            <li>
                        Bluebird Theater
            </li>
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            exact={true}
            to="/schedule/individual"
>
            <li>
                        Your Schedule
            </li>
</NavLink>,
        ],
        "className": "header-nav-list",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <li>
              All Venues
</li>,
            "exact": true,
            "to": "/schedule",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "All Venues",
            },
            "ref": null,
            "rendered": "All Venues",
            "type": "li",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <li>
              Ogden Theatre
</li>,
            "exact": true,
            "to": "/schedule/ogden",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Ogden Theatre",
            },
            "ref": null,
            "rendered": "Ogden Theatre",
            "type": "li",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <li>
              Gothic Theatre
</li>,
            "exact": true,
            "to": "/schedule/gothic",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Gothic Theatre",
            },
            "ref": null,
            "rendered": "Gothic Theatre",
            "type": "li",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <li>
              Bluebird Theater
</li>,
            "exact": true,
            "to": "/schedule/bluebird",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Bluebird Theater",
            },
            "ref": null,
            "rendered": "Bluebird Theater",
            "type": "li",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <li>
              Your Schedule
</li>,
            "exact": true,
            "to": "/schedule/individual",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Your Schedule",
            },
            "ref": null,
            "rendered": "Your Schedule",
            "type": "li",
          },
          "type": [Function],
        },
      ],
      "type": "ul",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
